examples/codegen/econf/spec.ini

```diff
@@ -58,11 +58,11 @@ gen/enum/type = Charset
 
 [format/#/trim]
 type = boolean
-default = true
```

Comment:
Is this not working?

Reply:
I didn't try it, the type plugin would have to be mounted of course. Whether that is automatically done by `spec-mount` I don't know.

Reply:
Yes, spec-mount should mount the type plugin if a "type" specifier is present. But if the default is transformed, I do not know: the key is added by the spec plugin, is the type plugin then executed later?